From: "Signed-off-by: NeilBrown" <neilb@suse.de>
Date: Mon, 31 Aug 2020 10:48:04 -0400
Subject: Convert remaining python scripts to python3
Origin: https://git.linux-nfs.org/?p=steved/nfs-utils.git;a=commit;h=d1683f356bd920d93f2db007902b2c46f97a2e49
Bug-Debian: https://bugs.debian.org/883194

nfs-utils contains 4 python scripts, two request
 /usr/bin/python3
in their shebang line, two request
 /usr/bin/python

Those latter two run perfectly well with python3 and as python2 is on the
way out, change them so they requrest /usr/bin/python3.

Signed-off-by: NeilBrown <neilb@suse.de>
Signed-off-by: Steve Dickson <steved@redhat.com>
---
 tools/mountstats/mountstats.py | 2 +-
 tools/nfs-iostat/nfs-iostat.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/mountstats/mountstats.py b/tools/mountstats/mountstats.py
index 1054f698c8e3..00adc96bafeb 100755
--- a/tools/mountstats/mountstats.py
+++ b/tools/mountstats/mountstats.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- python-mode -*-
 """Parse /proc/self/mountstats and display it in human readable form
 """
diff --git a/tools/nfs-iostat/nfs-iostat.py b/tools/nfs-iostat/nfs-iostat.py
index 50fd6a92977b..4f5e8a664ae8 100755
--- a/tools/nfs-iostat/nfs-iostat.py
+++ b/tools/nfs-iostat/nfs-iostat.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- python-mode -*-
 """Emulate iostat for NFS mount points using /proc/self/mountstats
 """
-- 
2.32.0

